### DISCUSSION TITLE
New User, no it experience

### DISCUSSION THREAD

**Discussion by henry18hn** [Ysurac/openmptcprouter#1597] [General]

**Comment by hairy-dog** [comment]
There does not seem to be a "complete beginner" guide.
This is what I think is needed:
1. Set up a VPS an internet server.  Instructions at https://github.com/Ysurac/openmptcprouter/wiki/Install-or-update-the-VPS
2. Set up an openmptcprouter - which is a computer on your network. I used a Raspberry Pi4. Instructions at https://github.com/Ysurac/openmptcprouter/wiki/Router-install
3. Switch off the DCHP servers on each of your modem/routers and set their IP addresses to different ranges  (192.168.10.1 and 192.168.11.1)
4. Configure your computer to be able to connect to 192.168.100.1 - I did this by changing the network adapter settings and scrolling down in the Networking tab to Internet Protocol Version 4 and changing the properties so the IP was 192.168.100.100 subnet 255.255.255.0 default gateway 192.168.100.1 - then I could access the new router and enter the settings into the wizard.
5. Put in all the details of the VPS and the modem/routers - see https://github.com/Ysurac/openmptcprouter/wiki/OpenMPTCProuter-configuration for instructions. Follow the bits under "Boxes" and "Wizard" - ignore the rest.
6. Then I switched my computer's network adapter settings back to "obtain automatically".
7. And it works! Everything is connected to the same ethernet network, with the modems on different subnet addresses, but the openmptcprouter device on the same IP range as all the other devices on the network (192.168.0.x in my case)

**Reply by henry18hn** [reply]
It took only a couple of tries, but I was able to get it up and running fairly easily. 
I am curious of why there's 4 vpn in the drop down box. Do I just try them out separately and pick one or should they all run together at the same time? I had to use the bypass to stream even though I paid for the services and I and the vps physical location are in the same region.
I am currently running on a RPi4b, wondering if I should upgrade my hardware.
I am using 2 4g modem currently. I habe heard of others setting up at a hard line locale to another locale wire hardline is not available. 
Thx for reply.

**Reply by hairy-dog** [reply]
There are countless settings that are almost entirely undocumented. My approach is to leave them well alone. 
Are you sure you had to use the bypass? When I tested, I thought I needed the bypass for Netflix, because it took a while to load with omr running. Then I tried again without it, and it was fine. 
Whether or not a Pi4 is powerful enough rather depends on what data speeds you are getting from your two 4G connections.  Between them, the best speed that I can manage from my connections is 75MB/sec. 
My Pi4 has no trouble coping with that, but if you are trying to combine two 175MB wan links, perhaps it might have a bottleneck in the ethernet port. The Pi4 CPU will cope easily: mine never gets even slightly busy.

**Reply by Ysurac** [reply]
Netflix, Amazon Video and Disney plus block many VPS IPs. RPI4 work without issue for up to 150-180Mb/s, can give better speed with good connections but a x86_64 based computer with AES-NI support always give a better result (but more expensive).

**Reply by hairy-dog** [reply]
My VPS is on a server in a standard hosting centre. I don't expect its used by anyone else as a VPS.
However, I can't tell. I only tried Netflix. Don't have or want accounts with the others.
